src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -144,6 +144,7 @@ module T = struct
           (* Invariant: this is the ledger after having applied all the transactions in
      *the above state. *)
     ; ledger: Ledger.attached_mask sexp_opaque
+    ; pending_coinbase_depth: int
```

Comment:
maybe include this in `Pending_coinbase.t` instead?

Reply:
I didn't push this down into `Pending_coinbase.t` because I was worried that we receive those values from the network, which would allow a malicious node to make us use the wrong `pending_coinbase_depth`.